app/models/ec_numbers.rb

```diff
@@ -0,0 +1,14 @@
+# == Schema Information
+#
+# Table name: ec_numbers
+#
+#  ec_number 		:string(12)       not null, primary key
+#  name	            :string(160)      not null
+#
+
+class EcNumber < ActiveRecord::Base
+  include ReadOnlyModel
+  attr_accessible nil
+
+  has_many :ec_cross_reference
+end
```

Comment:
add new line at the end of the file


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3366) by @bmesuere on Mon Oct 19 2015 at 14:40._